tasks/time-series/time-series-forecasting/XGB/src/runner.py

```diff
@@ -0,0 +1,150 @@
+import pandas as pd
+import numpy as np
+import io
+import requests
+from datetime import timedelta
+import xgboost as xgb
+from sklearn.model_selection import GridSearchCV
+from sklearn.metrics import *
+import matplotlib.pyplot as plt
+import mlflow
+import mlflow.sklearn
+
+
+class ForecastRunner(object):
+     def __init__(self, url, output_file, predicted_date, min_child_weight, colsample_bytree, max_depth, n_estimators):
+        self.url = url
+        self.output_file = output_file
+        self.predicted_date = predicted_date
+        self.min_child_weight = [float(i) for i in eval(min_child_weight)]
```

Comment:
Got error while reproducing results. Could you please help in resolving the problem. Following is the error.
I used readme example after removing --no-conda parameter

```
Traceback (most recent call last):
  File "main.py", line 14, in <module>
    model = ForecastRunner(url, output_file, predicted_date, min_child_weight, colsample_bytree, max_depth, n_estimators)
  File "/home/user/Downloads/ai-platform/tasks/time-series/time-series-forecasting/XGB/src/runner.py", line 19, in __init__
    self.min_child_weight = [float(i) for i in eval(min_child_weight)]
TypeError: eval() arg 1 must be a string, bytes or code object
```

Reply:
Updated. Can you try again?